test/unit/math/test_ad.hpp

```diff
@@ -1280,10 +1356,11 @@ void expect_ad_vectorized(const ad_tolerances& tols, const F& f, const T1& x1) {
  * @param f function to test
  * @param x argument to test
  */
-template <typename F, typename T>
+template <PromoteToComplex ComplexSupport = PromoteToComplex::Yes, typename F,
```

Comment:
[no action] This doesn't even look like a well-formed template parameter list!

Reply:
It seems odd but it does work!